src/locales/de/fields.json

```diff
@@ -277,6 +277,7 @@
   "Pending (waiting for activation)": "Ausstehend (wartet auf Aktivierung)",
   "Percentage of valid pixels": "Prozentsatz der gültigen Pixel",
   "Planet Labs Inc.": "Planet Labs Inc.",
```

Comment:
I guess the old Labels should be removed.

Reply:
I thought maybe that would have to wait for a dependency update of `@stac-utils/stac-fields`.

Reply:
Yes, but I'll wait to merge this until I've released stac-fields so you can already remove it here.